crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -99,14 +111,15 @@ impl SpendVisitor for MempoolVisitor {
             Condition::AggSigMe(_, _)
             | Condition::AggSigParent(_, _)
             | Condition::AggSigParentAmount(_, _)
-            | Condition::AggSigParentPuzzle(_, _) => {
+            | Condition::AggSigParentPuzzle(_, _)
+            | Condition::AggSigAmount(_, _)
+            | Condition::AggSigPuzzleAmount(_, _)
+            | Condition::AssertHeightRelative(_)
+            | Condition::AssertSecondsRelative(_) => {
```

Comment:
These two should not clear the `ELIGIBLE_FOR_DEDUP` flag, should they?

I believe you should also check `AssertEphemeral` here. It's considered a relative-timelock.

Also `ASSERT_BEFORE_SECONDS_RELATIVE`, `ASSERT_BEFORE_HEIGHT_RELATIVE`, `ASSERT_MY_BIRTH_HEIGHT` , `ASSERT_MY_BIRTH_SECONDS`. I think that's it

Reply:
Ephemeral and Before conditions are not affected by the fastforward - good catch with the birth height and birth seconds though, I've added those now